internal/helm/helm.go

```diff
@@ -185,10 +183,25 @@ func getChartVersionsFromOCIRepo(
 
 // versionsToSemVerCollection converts a slice of versions to a semver.Collection.
 // Any versions that cannot be parsed as SemVer are ignored.
-func versionsToSemVerCollection(versions []string) semver.Collection {
+func versionsToSemVerCollection(versions []string, isOCI bool) semver.Collection {
+	newSemver := semver.NewVersion
+	if isOCI {
+		// Image tags produced by Helm are STRICT SemVer, meaning that they
```

Comment:
Nit: Do we really want to say "Image tags" here? Applies in a few other places as well.

Totally non-blocking.

Reply:
Changed it to "OCI artifact tags".